### DISCUSSION TITLE
Is there anything wrong with this usage?

### DISCUSSION THREAD

**Discussion by Neleon** [deepspeedai/DeepSpeed#1672] [Q&A] [answered]
In stage_1_and_2.py, line 126:
    if not torch.cuda.is_available:
        raise SystemError("Cannot use fp16 without CUDA.")
My question is that is_available is a function, but here used as a property, is it OK?

**Comment by tjruwase** [comment, marked as answer]
This is surely a bug, as it should be a function call.  Thanks for reporting.
